graphics/src/Animation.cc

```diff
@@ -262,6 +262,8 @@ PoseAnimation::~PoseAnimation()
 {
   delete this->positionSpline;
   delete this->rotationSpline;
+  for (auto kf : this->keyFrames)
+    delete kf;
```

Comment:
When bringing this patch forward to the main branch, let's change these to all use smart pointers.  I don't think it's possible in a minor, because it would break ABI.